craft_application/application.py

```diff
@@ -134,7 +134,7 @@ def cache_dir(self) -> str:
         # xdg types: https://github.com/python/typeshed/pull/10163
         return save_cache_path(self.app.name)  # type: ignore[no-any-return]
 
-    def _configure_services(self, platform: str | None, build_for: str | None) -> None:
+    def _configure_services(self, build_for: str | None) -> None:
```

Comment:
rockcraft still needs the `platform` for the package service (it names the ROCK file)